source/documentation/common/install/proc-Deploying_the_Self-Hosted_Engine_Using_the_CLI.adoc

```diff
@@ -25,6 +25,11 @@ When the `he_pause_host` or `he_pause_before_engine_setup` variables are set to
 ** `after_add_host`
 ** `after_setup`
 
+[IMPORTANT]
```

Comment:
I suggest to move this to the first bullet in the Prerequisites, with the following rewording:
"Upgrade the appliance content to the latest product version before performing `engine-setup`.
To do this manually, pause the deployment using `he_pause_before_engine_setup` and perform a `dnf update`. To do this automatically, apply the `enginevm_before_engine_setup`  hook."

Reply:
@emarcusRH I can't get the syntax right. can you push a new commit with that change please? or let me know how exactly it should look like so it renders properly.
Thanks!